gangplank/ocp/filer.go

```diff
@@ -163,7 +168,10 @@ func (m *minioServer) kill() {
 	if m.cmd == nil {
 		return
 	}
-	_ = m.cmd.Process.Kill()
+	// Note the "-" before the processes PID. A negative pid to
+	// syscall.Kill kills the processes Pid group ensuring all forks/execs
+	// of minio are killed too.
+	_ = syscall.Kill(-m.cmd.Process.Pid, syscall.SIGKILL)
```

Comment:
It's not run in a pgroup today is it?  Won't this result in killing the current pgroup?

Bigger picture I think pgroups aren't very reliable; they've basically been replaced by cgroups - this is a bit related to e.g. https://github.com/openshift/machine-config-operator/pull/2259#discussion_r532904812

It'd seem more reliable if we send minio `SIGTERM` (graceful) when we exit, and then if it's done any forks on its own, either it uses pdeathsig (as we do) or it propagates the SIGTERM to its own children.

Reply:
(Or of course we try to run minio in its own container always)

Reply:
Just pushed a change that creates a `pgroup` so that this is safer. Also, switched to `SIGTERM`.